src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -749,21 +1117,38 @@ describe('LibraryServiceClient', function() {
       var expectedResponse = {};
 
       // Mock Grpc layer
-      client._addLabel = function(actualRequest, options, callback) {
-        assert.deepStrictEqual(actualRequest, request);
-        callback(null, expectedResponse);
-      };
+      client._addLabel = mockSimpleGrpcMethod(request, expectedResponse);
 
       client.addLabel(request, function(err, response) {
         assert.ifError(err);
         assert.deepStrictEqual(response, expectedResponse);
         done();
       });
     });
+
+    it('invokes addLabel with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var formattedResource = client.bookPath("[SHELF_ID]", "[BOOK_ID]");
+      var label = 'label102727412';
+      var request = {
+          resource : formattedResource,
+          label : label
+      };
+
+      // Mock Grpc layer
+      client._addLabel = mockSimpleGrpcMethod(request, null, error);
+
+      client.addLabel(request, function(err, response) {
+        assert(err instanceof Error);
+        assert.equal(err.code, FAKE_STATUS_CODE);
+        done();
+      });
+    });
   });
 
-  describe('getBigBookAsync', function() {
-    it('invokes getBigBookAsync without error', function(done) {
+  describe('getBigBook', function() {
+    it('invokes getBigBook without error', function(done) {
       var client = libraryV1.libraryServiceClient();
```

Comment:
This response of the method will be [wrapped](https://github.com/googleapis/gax-nodejs/blob/master/lib/longrunning.js#L94) by an operation so simply checking the response will not work. 

Maybe the most appropriate way would be a simple check like such:

```js
client.getBigBook(request, function(err, operation, response) {
    assert.ifError(err);
    assert.deepStrictEqual(operation.latestResponse, expectedResponse);
    assert.deepStrictEqual(response, expectedResponse);
    done();
  });
});
```